test/mock/scripts/session-error.ts

```diff
@@ -41,5 +41,7 @@ const bindings = basicBindings();
 runTest("sessionError", async (page: Page) => {
   await page.goto(devtoolsURL({ id: recordingId }));
   await installMockEnvironment(page, { graphqlMocks, messageHandlers, bindings });
-  await page.textContent("text=Unexpected session error");
+  await page.textContent(
+    "text=Something went wrong while replaying, we'll look into it as soon as possible"
+  );
```

Comment:
should this be it's own tiny PR fix to get this suite green?

Reply:
These tests are still super flaky sadly. They pass maybe 25% of the time locally.

Reply:
:/ that's sad. All the more reason to ship lots of small fixes and chip away at it.